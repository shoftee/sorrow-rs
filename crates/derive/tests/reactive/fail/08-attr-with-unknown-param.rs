use sorrow_derive::Reactive;

#[derive(Reactive)]
struct OuterTest {
    #[reactive(unexpected)]
    inner: InnerTest,
}

struct InnerTest {
    test_field: i32,
}

fn main() {}
