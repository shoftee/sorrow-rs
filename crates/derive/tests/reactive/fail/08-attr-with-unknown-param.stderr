error: invalid reactive attribute
 --> tests/reactive/fail/08-attr-with-unknown-param.rs:5:16
  |
5 |     #[reactive(unexpected)]
  |                ^^^^^^^^^^
